react/features/deep-linking/functions.js

```diff
@@ -52,9 +52,13 @@ export function generateDeepLinkingURL() {
 export function getDeepLinkingPage(state) {
     const { room } = state['features/base/conference'];
     const { launchInWeb } = state['features/deep-linking'];
+    const appScheme = interfaceConfig.APP_SCHEME;
```

Comment:
This can break mobile.

Reply:
in what way?

Reply:
There is no interfaceConfig on mobile, and I think this may be is imported in some native code.
Should be guarded like https://github.com/jitsi/jitsi-meet/blob/2f9fb64332ac2489a4a5bf80059848f81d79719f/react/features/invite/functions.js#L442

Reply:
fixed